packages/flutter/test/foundation/service_extensions_test.dart

```diff
@@ -716,12 +716,12 @@ void main() {
     expect(trace, contains('dart:core,Object,Object.\n'));
     expect(trace, contains('package:test_api/test_api.dart,::,test\n'));
     expect(trace, contains('service_extensions_test.dart,::,main\n'));
-  }, skip: isBrowser);
+  }, skip: isBrowser); // Reference needed
```

Comment:
@ferhatb is there an issue or comment I can add here?

Reply:
// Compilation trace is Dart VM specific and not supported in browsers.